course/page/base.py

```diff
@@ -780,6 +784,13 @@ def __init__(self, vctx, location, page_desc):
                         _("no title found in body or title attribute"))
                     % (location))
 
+        from markdown import markdown
+        from django.utils.html import strip_tags
+        title = strip_tags(markdown(title))
```

Comment:
Assuming `strip_tags` removes all HTML? If so, that seems fine. The title *is* fished out of the Markdown after all... Though I still don't fully understand the failure mode, as I noted.

Reply:
Both `**important**` and  `<b>important</b>`  are expected to be `important` in title.